Allowing users to ingest/report data (#135)
microsoft/ccf-app-samples: data-reconciliation-app/docs/adr/04-Authentication.md

- JWT Authentication
- Certificate Authentication

## Roles and Responsibilities

> By the way CCF is implemented, Members are capable of perform governance actions, whereas Users can interact only with the application endpoints.

Reviewer: ```suggestion
In CCF, Members are capable of performing governance actions, whereas Users can only interact with the application endpoints.
```

> For Data-Reconciliation App, Members can also interact with the application endpoints. For the application perspective, Members and Users have the same roles (both are capable of ingesting/reporting data and there is no distinction between them). For governance actions related to the Data-Reconciliation App (submitting and voting for proposals), again, only members will be able to interact with the governance-specific endpoints.

Reviewer: ```suggestion
In the Data-Reconciliation Application, Members can also interact with the application endpoints. From a application perspective, Members and Users have the same roles (both are capable of ingesting/reporting data and there is no distinction between them). For governance actions related to the Data-Reconciliation App (submitting and voting for proposals), again, only members will be able to interact with the governance-specific endpoints.
```
